scraping/scraping/start_scripts/amazon_eu.py

```diff
@@ -157,9 +161,12 @@ def female(country_code: str) -> List[dict]:
         "uk-apparel/Categories/Women/Sportswear/Track Jackets": ("JACKET", {"type": "SPORT"}),
         "uk-apparel/Categories/Women/Sportswear/Tracksuits": ("TRACKSUIT", {"type": "SPORT"}),
         "uk-apparel/Categories/Women/Sportswear/Trousers": ("PANTS", {"type": "SPORT"}),
+        "uk-apparel/Categories/Women/Suits & Blazers": "SUIT",
```

Comment:
I was not able to find this parent category in the json file, it always includes a child category. Not sure if this is somehow handled in the code. But anyway we do not want to assign parent categories that include different (GPC) subcategories to one GPC category. That's why I actually removed these already as part of 4b02bf43462e4d978a7e4a6af1e6576b7d48c0a9

What is the reason for adding them back again?

Reply:
Ah ok, that's nice! I was not aware of it. Thanks for clarifying this and adding a comment 👍 
I will try to check the category assignments tomorrow again, based on the new information.

Reply:
Just did a minor change for longsleeve tops in 0dbfd6532a8d92959f12cfd96117bbd8f1d822fe